api/v1/helpers/verbs/doPatch.js

```diff
@@ -68,6 +69,11 @@ function doPatch(req, res, next, props) {
 
   patchPromise
   .then((retVal) => {
+    if (props.modelName === 'Perspective') {
+      helper.validateFilterAndThrowError(retVal.get());
+
+    }
+
```

Comment:
the line needs to be there, since it needs the updated perspective, not only the request body. Ie. if aspectTags = [], a request to PATCH with aspectTagFilter='INCLUDE' should fail.